### PR TITLE
Use client build for azure test packages

### PR DESCRIPTION
**Pull request by scottn12**
## Description

Azure packages were recently moved back into the client release group, but the azure test pipelines will still try to pull the latest build from the azure release group. This PR changes this behavior to pull from the client release group.

### CODE REVIEW

**Review by sonalideshpandemsft — APPROVED** [review]

File: tools/pipelines/templates/include-test-real-service.yml [hunk 1]
```diff
@@ -106,12 +102,7 @@ jobs:
         ${{ if eq(variables.isTestBranch, 'true') }}:
           value: $(ado-feeds-internal) # Comes from the ado-feeds variable group
         ${{ else }}:
-          value: $(ado-feeds-dev) # Comes from the ado-feeds variable group
-      - name: artifactPipeline
-        ${{ if eq(parameters.downloadAzureTestArtifacts, true) }}:
-          value: Build - azure
-        ${{ else }}:
-          value: Build - client packages
+          value: $(ado-feeds-dev) # Comes from the ado-feeds variable group\
```
Comment (alexvy86) [inline tools/pipelines/templates/include-test-real-service.yml:105]:
```suggestion
          value: $(ado-feeds-dev) # Comes from the ado-feeds variable group
```

File: tools/pipelines/test-azure-frs.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Microsoft Corporation and contributors. All rights reserved.
 # Licensed under the MIT License.
 
-# test-azure-frs pipeline
+# test-real-service-e2e pipeline
```
Comment (alexvy86) [inline tools/pipelines/test-azure-frs.yml:4]: Was this intentional? Feels like it should stay the way it was.

**Review by alexvy86 — COMMENTED** [review]
There's one comment below, and a broader one: as far as I understand, the `Build - azure` pipeline is not used for `main` anymore, correct? Should we remove it as a trigger for this one?

PS: I see this got merged while I was reviewing 😄. I think at least one of the comments points out a bug, though.

**Comment by scottn12** [comment]
> PS: I see this got merged while I was reviewing 😄. I think at least one of the comments points out a bug, though.

@alexvy86, sorry about that 😅  These changes weren't enough anyway, and I'll be opening a follow up PR later today and I'll be sure to add you on to that one!

**Pull request merged**